src/components/HomePage/FAQSection.js

```diff
@@ -4,49 +4,58 @@ import { Typography, Link } from '@material-ui/core';
 import Container from '@material-ui/core/Container';
 import MuiExpansionPanel from '@material-ui/core/ExpansionPanel';
 import MuiExpansionPanelSummary from '@material-ui/core/ExpansionPanelSummary';
-import ExpansionPanelDetails from '@material-ui/core/ExpansionPanelDetails';
+import MuiExpansionPanelDetails from '@material-ui/core/ExpansionPanelDetails';
 
 import Add from '@material-ui/icons/Add';
 import Remove from '@material-ui/icons/Remove';
 
 const ExpansionPanel = withStyles(theme => ({
 	root: {
 		backgroundColor: theme.palette.background.default,
-		borderTop: '1px solid rgba(0, 0, 0, 1)',
-		borderBottom: '1px solid rgba(0, 0, 0, 1)',
+		borderTop: '1px solid black',
+		borderBottom: '1px solid black',
 		boxShadow: 'none',
 		'&:not(:last-child)': {
 			borderBottom: 0
 		},
-		'&:before': {
+		'&::before': {
 			display: 'none'
 		},
 		'&$expanded': {
-			margin: 'auto'
+			margin: 0
 		}
 	},
 	expanded: {}
 }))(MuiExpansionPanel);
 
-const ExpansionPanelSummary = withStyles({
+const ExpansionPanelSummary = withStyles(theme => ({
 	root: {
-		minHeight: 20,
+		minHeight: 'auto',
 		'&$expanded': {
-			minHeight: 20
+			minHeight: 'auto'
+		},
+		[theme.breakpoints.down('sm')]: {
+			paddingLeft: 8,
+			paddingRight: 8
```

Comment:
As discussed, this makes the page take up more space on the smallest devices that lack much display real estate. `ExpansionPanelDetails` is modified to match this.